features/polyExplorer/package.json

```diff
@@ -7,32 +7,38 @@
     "lint:fix": "eslint --fix src test scripts --ext .js --ext .jsx",
     "build": "rollup -c",
     "watch": "rollup --watch -c",
-    "test": "mocha && npm run cypress:run",
+    "test": "jest && npm run cypress:run",
     "polypedia:fetch": "node scripts/fetch-polypedia-data 162daa1",
     "polypedia:convert": "node scripts/convert-polypedia-data",
     "update-polypedia-data": "npm run polypedia:fetch && npm run polypedia:convert",
     "cypress:run": "cypress run"
   },
   "devDependencies": {
     "@babel/core": "^7.12.13",
+    "@babel/preset-env": "^7.16.11",
     "@polypoly-eu/podjs": "file:../../podjs",
     "@polypoly-eu/poly-look": "file:../../core/utils/poly-look",
     "@polypoly-eu/rollup-plugin-copy-watch": "file:../../core/utils/rollup-plugin-copy-watch",
-    "mocha": "^9.1.3",
     "react": "^16.13.1",
-    "react-dom": "^16.13.1",
-    "shelljs": "^0.8.5"
+    "react-dom": "^16.13.1"
   },
   "private": true,
   "dependencies": {
     "@polypoly-eu/silly-i18n": "file:../../core/utils/silly-i18n",
-    "@types/prop-types": "^15.7.3",
     "d3": "^7.0.0",
-    "lodash": "^4.17.21",
     "react-infinite-scroll-component": "^6.0.0",
     "react-router-dom": "^5.2.0",
-    "styled-components": "^5.3.0",
     "swiper": "^6.4.11"
   },
-  "type": "module"
+  "type": "module",
+  "jest": {
+    "verbose": true,
+    "testMatch": [
+      "**/test/*.js",
+      "!**/test/test*.js"
+    ],
+    "transform": {
+      "^.+\\.js$": "../../node_modules/babel-jest"
```

Comment:
Why is this necessary with Jest but not Mocha? Almost regardless of the question: Why don't we get rid of Jest and use Mocha everywhere if it requires such shenanigans? :D

Reply:
Well, jest is a framework, mocha is a test runner. The fact that it's a framework probably explains why you need to do that to fit all possible ways of including modules.
In purely practical terms, this was the only place left with mocha; some others were ported to jest some time ago, because at the time we started, it was the majority. Jest is also parallel when it can... In general I would say the best practice for testing has shifted from mocha to jest lately.
We are using anyway babel in most modules, so at this point that's almost exclusively a configuration issue.

Reply:
I'm on it, but, as anything related to babel and/or monorepos, not as easy as it seems...

Reply:
Also, [`jest`](https://github.com/facebook/jest/blob/3edccf64a6f46350d0e2688af79ae52a080ad9f4/examples/async/.babelrc.js) itself uses these shenanigans to configure examples. So it's likely there're no more options...

Reply:
But what do I know... Yep, there's always one way to do it...